modules/data-factory/factories/main.bicep

```
@@ -42,6 +42,9 @@ param gitCollaborationBranch string = 'main'
 @description('Optional. The root folder path name. Default is \'/\'.')
 param gitRootFolder string = '/'
 
+@description('Optional. The GitHub Enterprise Server host (prefixed with \'https://\'). . Only relevant for \'FactoryGitHubConfiguration\'. Default is \'\'.')
```

```suggestion
@description('Optional. The GitHub Enterprise Server host (prefixed with 'https://'). . Only relevant for 'FactoryGitHubConfiguration'.)
```

Note: This requires an update of the readme. You can simply run the `utilities/tools/Set-ModuleReadMe.ps1` utility, referencing the template.